test/shared_test/lib_battery_voltage_test.h

```diff
@@ -77,4 +77,16 @@ class voltage_vanadium_lib_battery_voltage_test : public lib_battery_voltage_tes
     }
 };
 
+class voltage_table_voltage_test : public lib_battery_voltage_test
+{
+protected:
+    void CreateModel(double dt_hr) {
+        std::vector<double> voltage_vals = {0, 1.7, 4, 1.7, 5, 1.58, 60, 1.5, 85, 1.4, 90, 1.3, 93, 1.2, 95, 1, 96, 0.9};
+        util::matrix_t<double> voltage_table(9, 2, &voltage_vals);
+        
+        cap = std::unique_ptr<capacity_lithium_ion_t>(new capacity_lithium_ion_t(10, 50, 95, 5, dt_hr));
+        model = std::unique_ptr<voltage_t>(new voltage_table_t(n_cells_series, n_strings, voltage_nom, voltage_table, R, dt_hr));
+    }
+};
+
```

Comment:
Do you mind combining the tests under this class with the `voltage_table_lib_battery_voltage_test` above?

Reply:
Why do we have two voltage_table test classes? https://github.com/NREL/ssc/blob/d3ba7ace8eea5b3d836f915b07f6cebe449fb44d/test/shared_test/lib_battery_voltage_test.h#L52

Reply:
I didn't notice the first one when writing tests. I combined them before pushing to develop: https://github.com/NREL/ssc/commit/4a573c3cb0b6727056e9e82990e03213718ebf25

Reply:
Sorry I missed that! Thanks